Cargo.toml

```diff
@@ -11,12 +11,15 @@ readme = "README.md"
 keywords = ["read", "password", "security", "pass", "getpass"]
 edition = "2018"
 
+[features]
+serde = ["dep:serde", "dep:serde_json"]
```

Comment:
FYI, this new `dep:` syntax prevents rpassword 7.0.0 from being using with rust 1.59 and older.  See https://github.com/rust-lang/cargo/issues/10623

Reply:
Oh right, good spot! I've added a note about this in the release on Github.